### PR TITLE
Support to display song books tab

### PR DESCRIPTION
**Pull request by madasamys**

### CODE REVIEW

File: app/src/main/java/org/worshipsongs/dao/SongBookDao.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        cursor.close();
+        return songBooks;
+    }
+
+    private SongBook cursorToSongBook(Cursor cursor)
+    {
+        SongBook songBook = new SongBook();
+        songBook.setId(cursor.getInt(0));
+        songBook.setName(parseName(cursor.getString(1)));
+        songBook.setPublisher(cursor.getString(2));
```
Comment (VigneshPalanisamy) [inline app/src/main/java/org/worshipsongs/dao/SongBookDao.java]: In songBook can we have a another variable for tamilName and we parse it default.

File: app/src/main/java/org/worshipsongs/domain/SongBook.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                '}';
+    }
+
+    @Override
+    public boolean equals(Object object)
+    {
+        if (object instanceof SongBook) {
+            SongBook otherObject = (SongBook) object;
+            EqualsBuilder equalsBuilder = new EqualsBuilder();
+            equalsBuilder.append(getName(), otherObject.getName());
```
Comment (VigneshPalanisamy) [inline app/src/main/java/org/worshipsongs/domain/SongBook.java]: can we include ID in equals

Comment (madasamys) [reply app/src/main/java/org/worshipsongs/domain/SongBook.java]: Usually we include unique field except primary key in equals method.

Comment (VigneshPalanisamy) [reply app/src/main/java/org/worshipsongs/domain/SongBook.java]: ok. i thought ID as song number!

**Pull request merged**